themes/cerberus/index.txt

```diff
@@ -13,8 +13,13 @@
 <% } %>
 
 <% if (locals.action) { %>
-<%- action.instructions %>
-<%- action.button.link %>
+<% if(locals.action.constructor !== Array){ %>
```

Comment:
Interesting, I didn't know that. When you say top of the file, do you mean line 1 above `<%- title %>` ? That will put a line break above the title.